subsys/ipc/ipc_service/backends/Kconfig.icmsg

```diff
@@ -1,10 +1,17 @@
 # Copyright (c) 2022 Nordic Semiconductor ASA
 # SPDX-License-Identifier: Apache-2.0
 
-if IPC_SERVICE_BACKEND_ICMSG
+config ICMSG
+	bool
+	select SPSC_PBUF
+	select SPSC_PBUF_USE_CACHE
+	help
+	  If icmsg library is to be included in the build
```

Comment:
If this is a library now, this should be moved to `subsys/ipc/ipc_service/backends/Kconfig`.

The idea is that `Kconfig.icmsg` is hosting only the configuration parameters for the backend while all the backends and libraries should be defined in the main `Kconfig`.

Reply:
I wasn't aware of this idea. I'll update accordingly.